.rollup.cjs

```diff
@@ -0,0 +1,14 @@
+const commonjs = require('@rollup/plugin-commonjs');
```

Comment:
I think it's preferable to stick to our more widely used stack and use webpack instead.